### ISSUE TITLE
After transferring a preset, I pressed the flashing preset button in the mode again and blacked out.

### ISSUE THREAD

**Issue by Jpro600k** [image-et-son/p600fw#114]
About alpha 14.2
After transferring a preset, I pressed the flashing preset button in the mode again and blacked out.
I think it would be kinder to switch the blinking to the RECORD button when the transfer is complete for leave the mode.
I think it makes more sense to press the record button to exit the mode.

**Comment by matrix12x** [comment]
That is normal in that mode. If you wait a little bit, the display lets you know it just did a bulk dump. 

Patch Management Mode is accessed by pressing "from tape" + Record.

Pressing preset in this mode initiates a bulk dump. When bilk dump is done, the display displays "presets dumped".

Pressing the preset number in this mode dumps that number preset and you get a message indicating the preset number that was dumped.

**Comment by Jpro600k** [comment]
It may not have been possible to convey that I responded to the blinking sensuously on the first use, however it is a personal impression. If I keep the current function and use it more times, I will be able to remember it.

**Comment by image-et-son** [comment]
Intuitively you would press a blinking button to leave the mode. In case of the patch management mode I simply ran out of natural options: making the record button blink is already reserved for storage mode. Pressing "From Tape" + "Preset" is used for loading the default patch (I needed two more option in the settings menu for 1) local on/off and 2) switching the panel layout) so I moved the default patch to "From Tape" + "Preset". It it is totally non-intuitive I could move the patch dump function to the record button so that pressing the blinking "Preset" can instead stop the patch mgmt mode.
